### PR TITLE
Updated Current Homepage to Conceptual Homepage Theme

### PR DESCRIPTION
**Pull request by Redblaze74**
- Changed README.md to redirect to main repository PDF in case of future updates of PDF
- Separated page.js stylizations into styles.css 
- Refined and added more code into both page.js and styles.css
- Included necessary icons and temporary images that will be used for other pages

### CODE REVIEW

**Review by aiharc1 — APPROVED** [review]
Nice work Alec. The basic fundamentals of page transitions are there

**Review by jusasin — APPROVED** [review]
The added functionality is a huge help, now we can work on adding design to the different features of the website. Great work!

**Review by junxjr — APPROVED** [review]
Everything looks great. I like the designs.

File: small-talk/app/homepage/page.js [hunk 1]
```diff
@@ -1,156 +1,71 @@
...
-
-                {/* creating a profile button, and direct to the profile page with ButtonStyling for the button */}
-                <HoverButton style={ButtonStyle} onClick={handleProfile}>Profile</HoverButton>
+                {/* creating a food button, and direct to the food page with ButtonStyling for the button */}
+                <HoverButton onClick={handleRedirect('/food')} iconImage="/img/food-tab.png" altText="Order Food" />
 
-                {/*creating a signout button, and direct to the login page with ButtonStyling for the button */}
-                <HoverButton style={ButtonStyle} onClick={handleSignOut}>Sign Out</HoverButton>
+                {/* creating a game button, and direct to the game page with ButtonStyling for the button */}
+                <HoverButton onClick={handleRedirect('/game')} iconImage="/img/games-tab.png" altText="Games" />
```
Comment (blake-gilmore) [inline small-talk/app/homepage/page.js:65]: Would love if we could implement a <menuButton> custom component that can take a list of redirects, icon image paths, and altText strings and map the list to a series of menu buttons. Then we can be sure to map that within a sidebar custom component as well to separate the functionality easier.

File: small-talk/app/homepage/page.js [hunk 2]
```diff
@@ -1,156 +1,71 @@
...
+                <button onClick={handleRedirect('/')}>
+                <img src="/img/logout-icon.png" alt="Notifications" style={{ height: '80px', marginTop: '45px', marginBottom: '10px', marginLeft: '1280px'}}/>
+                </button>
+                <button onClick={handleRedirect('/setting')}>
+                    <img src="/img/notif-icon.png" alt="Notifications" style={{ height: '80px', marginTop: '45px', marginBottom: '10px', marginLeft: '10px'}}/>
+                </button>
+                <button onClick={handleRedirect('/profile')}>
+                    <img src="/img/profile-temp.png" alt="Profile" style={{ height: '115px', marginTop: '10px', marginBottom: '10px', marginLeft: '20px' }}/>
+                </button>
+            </div>
```
Comment (blake-gilmore) [inline small-talk/app/homepage/page.js:44]: Currently shows in the vercel deployment:
image


To help debug this, when I zoom out I see:
image

And when I zoom in, I see:
image

**Review by blake-gilmore — CHANGES_REQUESTED** [review]

File: small-talk/app/homepage/page.js (hunk 1 above)
Comment (blake-gilmore) [reply small-talk/app/homepage/page.js:65]: This is being handled in issue #52

File: small-talk/app/homepage/page.js (hunk 2 above)
Comment (blake-gilmore) [reply small-talk/app/homepage/page.js:18]: Looks good
image

**Review by blake-gilmore — APPROVED** [review]

**Comment by blake-gilmore** [comment]
Looks great. Closes issue #10

**Pull request merged**